src/index.html

```diff
@@ -11,7 +11,187 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="groupDistance">
+        <legend>Personal information</legend>
+        <div>
+          <label>
+            Surname:
+            <input
+            class="inputDistance"
+            type="text"
+            id="surname"
+            name="surname"
+            autocomplete="off"
+            >
```

Comment:
```suggestion
            <input
              class="inputDistance"
              type="text"
              id="surname"
              name="surname"
              autocomplete="off"
            >
```
fix that formatting everywhere